CMakeLists.txt

```diff
@@ -5,9 +5,14 @@
 # Mozilla Public License Version 2.0
 
 # Set up the project.
-cmake_minimum_required( VERSION 3.11 )
+cmake_minimum_required( VERSION 3.21 )
```

Comment:
is this intended?

Reply:
Yeah that's to support `PROJECT_IS_TOP_LEVEL`

Reply:
> Aside from this, there's [PROJECT_IS_TOP_LEVEL](https://cmake.org/cmake/help/latest/variable/PROJECT_IS_TOP_LEVEL.html) from CMake 3.21

3.21 seems to be the minimum